### PR TITLE
common/naming: a helper utility to provide names for different parts of release-services

### PR DESCRIPTION
**Pull request by garbas**

### CODE REVIEW

**Comment by garbas** [comment]
@La0 i only implemented few name transformation, but I imagine as we go forward we would create few more.

File: src/common/naming/common_naming/__init__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import re
+
+
+LETTERS = re.compile('[^a-zA-Z]')
```
Comment (La0) [inline src/common/naming/common_naming/__init__.py]: Maybe use [string.acii_letters](https://docs.python.org/fr/3/library/string.html#string.ascii_letters) ?

Comment (garbas) [reply src/common/naming/common_naming/__init__.py]: How would you use `ascii_letts`?

Comment (La0) [reply src/common/naming/common_naming/__init__.py]: Nevermind, you use it to remove all non-letters occurences. The variable name confused me

**Comment by garbas** [comment]
@La0 can you review this PR again?

**Pull request merged**